frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -195,13 +197,7 @@ function BookInfo.getDocProps(ui, file, book_props, no_open_document, no_customi
         original_props = BookInfo.getBookProps(file, book_props, no_open_document)
         filepath = file
     end
-
-    local props = no_customize and original_props or BookInfo.customizeProps(original_props, filepath)
-    if ui then
-        -- if original title is empty, generate it as filename without extension
-        props.display_title = props.title or filemanagerutil.splitFileNameType(filepath)
-    end
-    return props
+    return no_customize and original_props or BookInfo.customizeProps(original_props, filepath)
```

Comment:
So, it is now BookInfo.customizeProps() that generates display_title - I guess you moved it there to avoid code duplication.
But it feels that it's not really customize'ation :) and should be there even if somewhere we use `no_customize` ?
Anyway, up to you.

Reply:
I think further polishing will be done in the next PR with custom metadata.
It's on the way.